lib/cn/cn_tree.c

```diff
@@ -1279,10 +1277,13 @@ cn_tree_lookup(
 
     pfx_hashing = kt->kt_len > tree->ct_pfx_len && node->tn_pfx_spill;
     first = true;
+    err = 0;
 
     rmlock_rlock(&tree->ct_lock, &lock);
     while (node) {
+		struct kvset_list_entry *le;
         bool yield = false;
+		u32 child;
```

Comment:
The indentation looks off

Reply:
Ahhh, my style selector was broken, was wondering why it was inserting tabs, thx!